tagstudio/src/qt/modals/drop_import.py

```diff
@@ -0,0 +1,79 @@
+import os
+import shutil
+import typing
+
+from PySide6.QtGui import QDropEvent, QDragEnterEvent, QDragMoveEvent
+from PySide6.QtWidgets import QMessageBox
+
+if typing.TYPE_CHECKING:
+    from src.qt.ts_qt import QtDriver
+
+
+def dropEvent(driver: "QtDriver", event: QDropEvent):
+    if event.mimeData().hasUrls():
+        if event.mimeData().urls()[0].isLocalFile():
```

Comment:
just a styling nit: it would be cleaner if the logic here would be reversed, so the code would be less indented:

```
if not event.mimeData().hasUrls():
    return

if not if event.mimeData().urls()[0].isLocalFile():
    return

# original code, but without extra 8 spaces
```